### PR TITLE
feat(cloudformation): import current cloudformation with usage documentation

### LINKED ISSUE #1: Import Cloudformation for AWS support access
## Requirements

- Import Cloudformation template from public documentation ([link](https://help.doit.com/docs/services/consulting-support/aws-access#cloudformation-template))
- Create usage documentation

### PR DESCRIPTION
**Pull request by mvaude**
Adding some naming refactors open to discussion

fixes #1

### CODE REVIEW

File: cloudformation/template.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >
+  This template deploys the IAM Policies and Roles
+  needed for AWS Ops. It takes a parameter to define
```
Comment (mvaude) [inline cloudformation/template.yml:4]: It looks like the parameter for the payer roles is not present anymore, doesn't it !?

Comment (mvaude) [inline cloudformation/template.yml:4]: `s/AWS Ops/DoiT Ops/`?

**Review by migueldelucasdoit — COMMENTED** [review]
@mvaude will try to include a CFN linter to check also this template.

File: cloudformation/template.yml (hunk 1 above)
Comment (migueldelucasdoit) [reply cloudformation/template.yml:4]: DoiT Ops makes sense to me

**Review by migueldelucasdoit — APPROVED** [review]

**Pull request merged**